doc/LABELS.md

```diff
@@ -22,7 +22,7 @@ Some of the most used labels are:-
 * ` testing ` - These issues are usually for adding `unit tests`, `integration tests` or any other tests for a particular feature/program.
 * ` outreach ` - The outreach issues involve community involvement and helping people who're stuck somewhere.
 * Some issues have been labeled with ` summer-of-code ` , ` outreachy ` , ` first-timers-only `, ` gci-candidate ` and ` rgsoc ` which mean that these issues have been reserved for students who're participating in these events.
-* `brainstorm` - Issues which need discussion and requirements needs to be elicitated
+* `brainstorm` - "Issues that need discussion and requirements need to be elucidated"
```

Comment:
Looks good, except that the quotes don't need to be included here (to be consistent with the other descriptions).